### ISSUE TITLE
Medusa Log issue

### ISSUE THREAD

**Issue by saitoh183** [pymedusa/Medusa#3611]
### Before submitting your issue:

Enable debug logging in Medusa settings, reproduce the error (be sure to disable after the bug is fixed)

**Branch/Commit:**
**OS:**
**What you did:** Dont really know when it started
**What happened:** Logs with Debug look like below and in GUI there are no info/warning/error logs
**What you expected:**
**Logs:**
```
11e**********-9d29-a0**********69fbcd**********8**********)
2018-01-16 0**********:21:18 DEBUG    SHOWQUEUE-REFRESH :: [e1628********************] Image already exists, not downloading
2018-01-16 0**********:21:18 INFO     SHOWQUEUE-REFRESH :: [e1628********************] Cache check completed
2018-01-16 0**********:21:18 DEBUG    SHOWQUEUE-REFRESH :: [e1628********************] Looking up XEM scene mapping for show ID **********2908********** on TVDBv2
2018-01-16 0**********:21:18 DEBUG    SHOWQUEUE-REFRESH :: [e1628********************] GET URL: http://thexem.de/map/havemap?origin=tvdb [Status: 200]
2018-01-16 0**********:21:18 DEBUG    SHOWQUEUE-REFRESH :: [e1628********************] User-Agent: Medusa/0.1.2********** (Linux; **********.1**********.0-1**********2-generic; **********28b**********d06-fa**********1-11e**********-9d29-a0**********69fbcd**********8**********)
2018-01-16 0**********:21:18 DEBUG    SHOWQUEUE-REFRESH :: [e1628********************] No XEM data for show ID **********2908********** on TVDBv2
```
Why do my logs look like this and also they dont appear in on the webpage

![image](https://user-images.githubusercontent.com/1534197/34971322-c844b8d2-fa47-11e7-8086-8bb55c6b1a1a.png)

**Comment by labrys** [comment]
You have an overly simplistic password or username set somewhere.  When filtering known usernames and passwords from the logs its mangling the date time and commit hash in the loglines which is preventing the log filter from properly processing them for the UI.  Please set your username/passwords to something more secure.

**Comment by labrys** [comment]
Somewhere you have an apikey, password, username, or something that is the value `5`.  You also have the value `3` as another apikey, password, or username.

**Comment by saitoh183** [comment]
@labrys 

Thanks...the issue was that for some reason the torrent categories were messed up. I had added them and they were fine (5040,5030) but when i went to check a few were now showing one number per line
```
5
0
3
0
5
0
4
5
```

**Comment by p0psicles** [comment]
Can you go into a little bit more detail? What is your provider setup?

**Comment by saitoh183** [comment]
@p0psicles 

It seemed to be all the newznab ones that didnt save correctly even if they looked fine when i hit saved.

**Comment by heymoe** [comment]
Weird, I was having the same issue with my logging.

The custom search provider I setup had the category IDs displayed like @saitoh183 noted above and when I looked at the config.ini file, the config entry looked like this:

`custom_cat_ids = 5, 0, 3, 0, "", "", 5, 0, 4, 0`

I went back and refreshed and then re-added the 2 categories and they got added back to the config.ini like this:

`custom_cat_ids = 5030, 5040`

After a restart, my logs look normal again.

I have no clue how or why the categories got added like the did initially but everything seems happy now in log land.
